pkg/genericapiserver/genericapiserver_test.go

```diff
@@ -311,6 +313,29 @@ func TestDiscoveryAtAPIS(t *testing.T) {
 
 	assert.Equal(0, len(groupList.Groups))
 }
+*/
+
+func TestCORSRequest(t *testing.T) {
+	master, etcdserver, _, _ := newMaster(t)
+	defer etcdserver.Terminate(t)
+
+	server := httptest.NewServer(master.HandlerContainer.ServeMux)
```

Comment:
I think this should be `server := httptest.NewServer(master.Handler)` or `server := httptest.NewServer(master.InsecureHandler)`